lite/examples/object_detection/android/app/src/main/java/org/tensorflow/lite/examples/detection/tflite/TFLiteObjectDetectionAPIModel.java

```diff
@@ -207,12 +207,15 @@ public static Classifier create(
       // in label file and class labels start from 1 to number_of_classes+1,
       // while outputClasses correspond to class index from 0 to number_of_classes
       int labelOffset = 1;
-      recognitions.add(
-          new Recognition(
-              "" + i,
-              labels.get((int) outputClasses[0][i] + labelOffset),
-              outputScores[0][i],
-              detection));
+      int labelIndex = (int) outputClasses[0][i] + labelOffset;
+      if(labelIndex >= 0 && labelIndex < labels.size()) {
```

Comment:
This seems like a bug in the model? In any case, we should log an error in this case with some details about the labelIndex and score.

Nit: space after if ("if (labelIndex >= 0...)").